client/components/TestRun/index.jsx

```diff
@@ -671,18 +672,26 @@ class TestRun extends Component {
             );
 
             if (!this.state.runComplete) {
-                testContent = this.renderTest({
-                    key: `${test.id}/${this.state.currentTestIndex}`,
-                    run,
-                    test,
-                    testIndex: this.state.currentTestIndex,
-                    git_hash,
-                    at_key,
-                    userId,
-                    testerId: openAsUser || userId
-                });
+                testContent = this.renderTest(
+                    {
+                        key: `${test.id}/${this.state.currentTestIndex}`,
+                        run,
+                        test,
+                        testIndex: this.state.currentTestIndex,
+                        git_hash,
+                        at_key,
+                        userId,
+                        testerId: openAsUser || userId
+                    },
+                    heading
+                );
             } else {
-                content = <div>Tests are complete.</div>;
+                content = (
+                    <div>
+                        {heading}
```

Comment:
We were losing the heading when the tests were completed, so I added it to this content section.